src/Registrator.jl

```diff
@@ -3,14 +3,13 @@ module Registrator
 using UUIDs, LibGit2, DataStructures
 
 import Base: PkgId
-import Pkg: Pkg, TOML, GitTools
 
 const DEFAULT_REGISTRY = "https://github.com/JuliaRegistries/General"
 const REGISTRIES = Dict{String,UUID}()
 
 include("slack.jl")
 include("builtin_pkgs.jl")
-include("register.jl")
+include(joinpath("regedit", "RegEdit.jl"))
```

Comment:
```suggestion
include("regedit/RegEdit.jl")
```

Reply:
What, why?

Reply:
Looks a million times nicer.

Reply:
I like it better with the slash as well; is it guaranteed to be valid though?

Reply:
Yes, https://github.com/JuliaLang/julia/pull/26656